packages/quill/src/app/components/TxHistory/TxQueueCard.tsx

```diff
@@ -23,34 +23,43 @@ export const TxQueueCard = ({ item, first }: { item: TxQueueEntry; first: boolea
   const handleSelect = () => dispatch(selectTransaction(item));
 
   return (
-    <Box pt={first ? '0' : '16px'} pb="16px">
-      <Banner
-        type="clear"
-        label={translateRaw('HOME_TX_RESULT_WAITING')}
-        badge={first && translateRaw('NEW')}
-      />
-      <Box variant="horizontal-start" mt="2">
-        <FromToAccount
-          sender={{ address: tx.from, label: currentAccount?.label }}
-          recipient={tx.to && { address: tx.to, label: recipientAccount?.label }}
-          nonce={tx.nonce}
-        />
+    <Box
+      data-testid="pending-tx"
+      mt={first ? '0' : '16px'}
+      p="3"
+      sx={{ boxShadow: '0px 3px 6px rgba(0, 0, 0, 0.07)', borderRadius: 'banner' }}
+    >
+      <Flex variant="horizontal-start">
+        {first && (
+          <Badge type="clear" mr="1">
+            {translateRaw('NEW')}
+          </Badge>
+        )}
+        <Banner type="clear" m="0" p="0" pb="0" label={translateRaw('HOME_TX_RESULT_WAITING')} />
```

Comment:
Is `pb="0"` needed if `p="0"` is set?

Reply:
Yeah, in this case it is needed to overwrite the value set in the UI lib